test.tex

```diff
@@ -91,14 +89,16 @@
 
 \end{lilypond}
 
-\lilypond[staffsize=24]{\relative c'' { c b a g f e d c }}
+\lilypond[relative]{
+{ c d e f g a b c } \addlyrics{do ré mi fa sol la si do.}
+}
 
 Ce fragment :
 
-\lilypond{a' b' c'}
+\lilypond{a' b' c''}
 
 sera écrit plus petit en note de bas de
-page\footnote{\lilypond[fragment]{a' b' c'}}.
+page\footnote{\lilypond[relative=2]{a b c}}.
```

Comment:
Just to be clear: We haven't defined any `fragment` option yet, it's just that unknown local options don't raise errors, isn't it?

Reply:
For now, really, `fragment` is the default behaviour of `lyluatex` for `ly` and `\lily`. It could be discussed (as it isn't the default behaviour of `lilypond-book`), but it only raises errors when someone tries to use a `\score` block; IMHO, when you need it, it's time to factor out your code to put in within an external file.
That said, I plan to implement `nofragment`, so that if one really wants to do that, he can.

Reply:
I'll start working on `nofragment` as soon as this one is merged, to avoid conflicts.